### DISCUSSION TITLE
I'm making an app like TweetDeck. Is code like this okay?

### DISCUSSION THREAD

**Discussion by aoisensi** [rrousselGit/riverpod#992] [Q&A] [answered]
```dart
static final accounts = <String, ChangeNotifierProvider<Account>>{};
```
I just want to know if I can do something like `Map<String, Provider>` in Riverpod, or if I can create more than 100 `Provider`s.
I'm Flutter noob.
Please teach me other good solution if you have.
// sorry for bad english

**Comment by rrousselGit** [comment, marked as answer]
Avoid doing that.  You probably want to use `family` instead

**Reply by aoisensi** [reply]
Thank you for quick response.
I don't understand about `family`.
Japanese Riverpod commentary articles (blog, qiita, zenn, etc...) saying only "By using `.famliy`, `Provider` can accept arguments.".
(I cannot read complex English!)
Is it build `Provider`s like key-value store?

**Reply by aoisensi** [reply]
I found good example.
https://github.com/mono0926/riverpod_example/blob/main/lib/pages/counters_page/counter_provider.dart
I try this.
Thank you.

Can I send 1 coffee for you? (I want to tip once)

**Reply by aoisensi** [reply]
This is difficult for me ...
